Classes/Settings/SettingsAccountsViewController.swift

```diff
@@ -53,7 +53,15 @@ final class SettingsAccountsViewController: UITableViewController, GitHubSession
                     case .failure:
                         self?.handleError()
                     case .success(let user):
-                        self?.finishLogin(token: token, authMethod: .pat, username: user.data.login)
+                        let userPreviouslyAuthenticated = self!.sessionManager.userSessions.contains {
```

Comment:
Let's not force-unwrap `self` here maybe?

Reply:
Yea, good idea.  I'll update this.